src/Administration/Resources/app/administration/src/module/sw-settings/page/sw-settings-index/index.js

```diff
@@ -56,9 +74,57 @@ export default {
                 {},
             );
         },
+
+        /**
+         * @deprecated tag:v6.8.0 - Will be removed without replacement
+         */
+        shouldShowBanner() {
+            return !this.isLoading && !this.isUnitBannerHidden;
+        },
     },
 
     methods: {
+        /**
+         * @deprecated tag:v6.8.0 - Will be removed without replacement
+         */
+        async loadUserConfig() {
+            try {
+                const response = await this.userConfigService.search(['setting.unitBanner']);
```

Comment:
```suggestion
                const { hideMigratingScalesUnitBanner } = await this.userConfigService.search(['hideMigratingScalesUnitBanner']);
                
                this.hideMigratingScalesUnitBanner = hideMigratingScalesUnitBanner ?? false;
```

Reply:
I don't think we should get the user config for this key when visiting the module. The renaming banner should be visible by default.